whylabs_toolkit/monitor/models/analyzer/analyzer.py

```diff
@@ -23,6 +24,8 @@
 )
 from .targets import ColumnMatrix, DatasetMatrix
 
+CRON_REGEX = "^(0|\d) (\d+|\d+-\d+|\/\d+)(,\d+|\d+-\d+|\/\d+)* (\*|\d+|\d+-\d+|\/\d+) (\*|\d+|\d+-\d+|\/\d+) (\*|\d+|\d+-\d+|(1|2|3|4|5|6|0)(,\d+)*)$"
+
```

Comment:
seems like this only supports comma format for days - is it not valid for the other fields too?

Reply:
I prefer not to include a new dependency, so i've changed everything to be checked for with python. see if it's enough for now.